src/main/java/dev/jbang/spi/IntegrationManager.java

```diff
@@ -267,6 +278,7 @@ public static void main(String... args) {
 		IntegrationInput input = parser.fromJson(new InputStreamReader(System.in), IntegrationInput.class);
 		ClassLoader old = Thread.currentThread().getContextClassLoader();
 		PrintStream oldout = System.out;
+		Util.setVerbose(input.verbose);
```

Comment:
shouldn't this not be reversed afterwards?

Reply:
No, in the case JBang itself is running the code this line is a no-op (it will be setting it to its current value) while in the case of external code the app will exit afterwards anyway.